docker-compose.yml

```diff
@@ -93,6 +93,17 @@ services:
       - /bin/entrypoint.sh
     command: ["run"]
```

Comment:
Spricht etwas dagegen, für beide Zwecke eine Konfiguration zu nutzen, also die Anpassungen von cypress-gui direkt in cypress zu haben? Der default-command kann ja wie gehabt  run bleiben, amn kann dann aber einfach auch open angeben…

Reply:
Stimmt, finde ich gar keine schlechte Idee. Ich glaube docker-compose failt nicht einmal falls beim Volume mounting das Source File / Dir auf dem Host nicht existiert (oder?)

Reply:
nein, er erstellt dann einfach ein ordner…

Reply:
Ja, finde ich auch ok